cmd/root.go

```diff
@@ -25,15 +25,16 @@ var rootCmd = &cobra.Command{
 }
 
 func init() {
-	if os.Getenv("GITHUB_API_TOKEN") == "" {
-		log.Fatalf("GITHUB_API_TOKEN env var is not set. In order to use microplane, create a token (https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) then set the env var.")
+	if os.Getenv("GITHUB_API_TOKEN") == "" && os.Getenv("GITLAB_API_TOKEN") == "" {
+		log.Fatalf("GITHUB_API_TOKEN or GITLAB_API_TOKEN env var is not set. In order to use microplane, create a token (https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) then set the env var.")
 	}
 
 	rootCmd.PersistentFlags().StringP("repo", "r", "", "single repo to operate on")
 	rootCmd.AddCommand(cloneCmd)
 	rootCmd.AddCommand(docsCmd)
 	rootCmd.AddCommand(initCmd)
 
+	initCmd.Flags().StringVarP(&repoProviderFlag, "provider", "p", "", "Git Repository Provider: github or gitlab")
```

Comment:
nit: for readability, group this into a block in the same way done below for the other commands